loom/graphics/gfxTexture.cpp

```diff
@@ -277,6 +282,8 @@ void bitmapExtrudeRGBA_c(const void *srcMip, void *mip, int srcHeight, int srcWi
 
 TextureInfo *Texture::load(uint8_t *data, uint16_t width, uint16_t height, TextureID id)
 {
+	id &= TEXTURE_ID_MASK;
```

Comment:
Actually, I don't think so, since we only want to prevent drawing invalid textures and we probably don't want to prevent loading new textures using outdated ids (since we're going to update them anyway).